src/pe/mod.rs

```diff
@@ -86,6 +89,7 @@ impl<'a> PE<'a> {
         let mut libraries = vec![];
         let mut debug_data = None;
         let mut exception_data = None;
+        let mut certificates = vec![];
```

Comment:
nitpick:
```suggestion
        let mut certificates = Default::default();
```

certificates field is defined as a `CertificateDirectoryTable` :
```rust
pub type CertificateDirectoryTable<'a> = Vec<CertificateAttribute<'a>>;
// [...]
struct PE {
// [...]
    pub certificates: certificate_table::CertificateDirectoryTable<'a>,
}
```
Thus, it's not a inherently a `Vec` (although it's a type alias)

(this is equivalent: https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Default-for-Vec%3CT%2C%20Global%3E)